### PR TITLE
Add v5e tests for Pax

### PR DESCRIPTION
**Pull request by RissyRan**
# Description

Add v5litepod tests for Pax.
* Resources are in project `tpu-prod-env-automated`

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...
Upload to Airflow and test.

**List links for your tests (use go/shortn-gen for any internal link):** ...
Test: [link](https://screenshot.googleplex.com/9RuBPKXZvEGiy2T) - unfortunately, it's failing at `Could not find a version that satisfies the requirement flax (unavailable) (from praxis) (from versions: 0.1.0rc1,...` - same error as now

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: configs/xlml/pax/solutionsteam_pax_supported_config.py [hunk 1]
```diff
@@ -69,13 +69,18 @@ def get_setup_cmds(
     raise RuntimeError(f"Please specify set up cmds for: {pax_version.value}.")
 
 
-def get_runtime_version(pax_version: PaxVersion) -> str:
-  if pax_version is PaxVersion.STABLE:
-    return vm_resource.RuntimeVersion.TPU_VM_V4_BASE.value
-  elif pax_version is PaxVersion.NIGHTLY:
-    return vm_resource.RuntimeVersion.TPU_UBUNTU2204_BASE.value
+def get_runtime_version(pax_version: PaxVersion, tpu_version: str) -> str:
+  if tpu_version == "5litepod":
```
Comment (gkroiz) [inline configs/xlml/pax/solutionsteam_pax_supported_config.py:73]: why are we using the 5litepod notation here? Shouldn't it be 5e?

File: dags/xlml/solutionsteam_flax_latest_supported.py
Comment (gkroiz) [inline dags/xlml/solutionsteam_flax_latest_supported.py:1]: Maybe this should be in a different PR? Or the PR should be adjusted to also reflect flax test changes, not just pax.

File: configs/xlml/pax/solutionsteam_pax_supported_config.py (hunk 1 above)
Comment (RissyRan) [reply configs/xlml/pax/solutionsteam_pax_supported_config.py:73]: No, when creating the v5e, it's still v5litepod. The change starts from v5p.

File: dags/xlml/solutionsteam_flax_latest_supported.py
Comment (RissyRan) [reply dags/xlml/solutionsteam_flax_latest_supported.py:1]: v5 network variables were defined in flax only. I think it's better we move out in a common file that can be re-used in both flax, pax, and future frameworks that onboards with v5. That's the reason I included here. I don't see much value that we separate them into 2 PRs. HDYT?

Comment (gkroiz) [reply dags/xlml/solutionsteam_flax_latest_supported.py:1]: makes sense, we can leave as is

**Review by gkroiz — APPROVED** [review]
